mvn-build-all-targets.txt

```diff
@@ -6,13 +6,18 @@ terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0208
 terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0208kanji
 terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0213kanji
 terasoluna-gfw-validator
+terasoluna-gfw-dependencies-common
```

Comment:
- [ ] Are these sub module necessary? (I think `terasoluna-gfw-dependencies` manage to build them.)